microservice/frontend/catalog/debian/data/etc/nginx/sites-available/shop.microservice.io.1

```diff
@@ -0,0 +1,14 @@
+server {
+  listen   81; ## listen for ipv4; this line is default and implied
+  #listen   [::]:80 default ipv6only=on; ## listen for ipv6
+
+  root /usr/share/shop_A/frontend/catalog/html;
```

Comment:
@luisgtz - good progress - but it will need to work with Jenkins before we merge. What happens is that shop_a and shop_b directories are only created for by the cart microservice debian package (as per [this](https://github.com/devops-dojo/the-app/commit/51e23fd11650fe75700862cfb9a0e2d4765e40a7)). Maybe adding the same `copy into` to the frontend catalog debian package will be enough.